devicesetup/src/main/java/io/particle/android/sdk/devicesetup/ParticleDeviceSetupLibrary.java

```diff
@@ -26,10 +26,14 @@
      */
     public interface DeviceSetupCompleteContract {
 
-        /** The BroadcastIntent action sent when the device setup process is complete. */
+        /**
```

Comment:
Could you kill all the comment auto-formatting?  I'd like to keep one-liner comments/docs actually one line. ;)